tests/blackbox/extensions/action-verify-create/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "action-verify-create",
+	"version": "0.0.0",
+	"private": true,
+	"keywords": [
+		"directus",
+		"directus-extension",
+		"directus-custom-hook"
+	],
```

Comment:
Let's reduce to the props that are actually required, so we don't have to keep these files in "sync"

```suggestion
```

Reply:
Very true! Thanks!